charts/operator-wandb/charts/weave-trace/templates/_helpers.tpl

```diff
@@ -4,19 +4,19 @@
 Expand the name of the chart.
 */}}
 {{- define "weaveTrace.name" -}}
-{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- default "weave-trace" .Values.nameOverride | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
 {{/*
 Create a default fully qualified name for weave-trace. (Should be something like wandb-weave-trace)
 We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
 If release name contains chart name it will be used as a full name.
 */}}
-{{- define "weaveTrace.fullname" -}}
+{{- define "weave-trace.weaveTrace.fullname" -}}
 {{- if .Values.fullnameOverride }}
 {{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
 {{- else }}
-{{- $name := default .Chart.Name .Values.nameOverride }}
+{{- $name := default "weave-trace" .Values.nameOverride }}
```

Comment:
why are we hard coding this?

Reply:
This function is now called in the app server context so it was using the wrong name when called there. There's probably a better way for me to do it.

Reply:
Whats do you mean by app server context?

Reply:
technically, you should not be calling a template across charts